src/k8s/api/v1/types.go

```diff
@@ -82,7 +82,6 @@ func (c ClusterStatus) String() string {
 		result.WriteString("k8s is ready.")
 	} else {
 		result.WriteString("k8s is not ready.\n")
```

Comment:
consistent whitespace
```suggestion
		result.WriteString("k8s is not ready.")
```